website/src/components/header.tsx

```diff
@@ -0,0 +1,184 @@
+/* This example requires Tailwind CSS v2.0+ */
+import { Popover, Transition } from "@headlessui/react";
+import {
+  Bars3Icon,
+  MagnifyingGlassIcon,
+  XMarkIcon,
+} from "@heroicons/react/24/outline";
+import { Fragment } from "react";
+
+const navigation = {
+  main: [
+    { name: "Docs", href: "/docs" },
+    { name: "Blog", href: "/blog" },
+    { name: "Team", href: "/team" },
+  ],
+  social: [
+    {
+      name: "Discord",
+      href: "/",
+      icon: "/img/social/discord.svg",
+    },
+    {
+      name: "Twitter",
+      href: "/",
+      icon: "/img/social/twitter.svg",
+    },
+    {
+      name: "Github",
+      href: "/",
+      icon: "/img/social/github.svg",
+    },
+  ],
+};
```

Comment:
@functionless/functionless-team these should come from something central.

Reply:
Will add to docusaurus config